scipy/optimize/tests/test_minpack.py

```diff
@@ -517,6 +517,131 @@ def test_maxfev_and_bounds(self):
         assert_allclose(popt1, 2, atol=1e-14)
         assert_allclose(popt2, 2, atol=1e-14)
 
+    def test_curvefit_simplecovariance_nojacobian(self):
```

Comment:
I thought the idea of unit tests was to test one functionality at once, so that it is easy to debug which one failed. I can easily move them to one function, but I had kept it that way on purpose. Please confirm that this is the right thing to do.